target/linux/mediatek/dts/mt7981b-mercusys-mr80x-v3.dts

```diff
@@ -0,0 +1,328 @@
+// SPDX-License-Identifier: GPL-2.0
+
+/dts-v1/;
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+#include "mt7981.dtsi"
+/ {
+	model = "MERCUSYS MR80X v3";
+	compatible = "mercusys,mr80x-v3", "mediatek,mt7981";
+
+	aliases {
+		serial0 = &uart0;
+	};
+
+	chosen {
+		stdout-path = "serial0:115200n8";
+	};
+
+	memory {
+		reg = <0 0x40000000 0 0x10000000>;
+	};
+
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
+		poll-interval = <100>;
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&pio 1 GPIO_ACTIVE_LOW>;
+			linux,input-type = <1>;

```

Comment:
Please use the `EV_KEY` macro instead. Or, `EV_KEY` is a default type of `"gpio-keys"` (and `"gpio-keys-polled"`) and this can be removed.